src/Stackla/Api/Tile.php

```diff
@@ -627,7 +627,7 @@ public function getByGuid($id)
 
     public function getByStacklaFeedId($id)
     {
-        return parent::getById('sta_feed_id:'.$id);
+        return parent::getById('guid:'.$id);
```

Comment:
Hi @helman, ok I'm fixing it. However, in the Stackla doc @guid is describe as an alias og sta_ @feed_id.